waku/common/rate_limit/setting.nim

```diff
@@ -17,3 +40,101 @@ func `$`*(t: RateLimitSetting): string {.inline.} =
       "no-limit"
     else:
       $t.volume & "/" & $t.period
+
+proc translate(sProtocol: string): RateLimitedProtocol =
+  if sProtocol.len == 0:
+    return GLOBAL
+
+  case sProtocol
+  of "global":
+    return GLOBAL
+  of "storev2":
```

Comment:
I wonder if we should instead use the `const` items that we already have, such as:
https://github.com/waku-org/nwaku/blob/148a48a78c6d51c39f4913729b8674f834043487/waku/waku_store_legacy/common.nim#L7